src/creds.ts

```diff
@@ -1,260 +1,65 @@
+  const credentials = {
+    ...await getCredentials(filePath),
+    [machineName]: {
       login,
       password,
       region,
-    } as NetrcMachine
-
-    // Serialize machines back into .netrc format
-    const newContent = serializeNetrcMachines(machines)
+    },
+  }
 
-    // Write the updated content back to the .netrc file
-    await writeFile(filePath, newContent, {
-      mode: 0o600, // Set file permissions
-    })
+  try {
+    await saveToFile(filePath, JSON.stringify(credentials, null, 2), { mode: 0o600 })
 
     konsola.ok(`Successfully added/updated entry for machine ${machineName} in ${chalk.hex(colorPalette.PRIMARY)(filePath)}`, true)
   }
   catch (error) {
-    throw new FileSystemError('invalid_argument', 'write', error as NodeJS.ErrnoException, `Error adding/updating entry for machine ${machineName} in .netrc file`)
+    throw new FileSystemError('invalid_argument', 'write', error as NodeJS.ErrnoException, `Error adding/updating entry for machine ${machineName} in credentials.json file`)
   }
 }
 
-// Function to remove an entry from the .netrc file asynchronously
-export const removeNetrcEntry = async (
-  machineName: string,
-  filePath = getNetrcFilePath(),
-) => {
-  try {
-    let machines: Record<string, NetrcMachine> = {}
+export const removeCredentials = async (machineName: string, filepath: string = getStoryblokGlobalPath()) => {
+  const filePath = join(filepath, 'credentials.json')
+  const credentials = await getCredentials(filePath)
 
-    // Check if the file exists
-    try {
-      await access(filePath)
-      // File exists, read and parse it
-      const content = await readFile(filePath, 'utf8')
-      machines = parseNetrcContent(content)
-    }
-    catch {
-      return
-    }
+  if (credentials[machineName]) {
+    delete credentials[machineName]
 
-    if (machines[machineName]) {
-      // Remove the machine entry
-      delete machines[machineName]
-      // Serialize machines back into .netrc format
-      const newContent = serializeNetrcMachines(machines)
-
-      // Write the updated content back to the .netrc file
-      await writeFile(filePath, newContent, {
-        mode: 0o600, // Set file permissions
-      })
+    try {
+      await saveToFile(filePath, JSON.stringify(credentials, null, 2), { mode: 0o600 })
 
-      konsola.ok(`Successfully removed entry from ${chalk.hex(colorPalette.PRIMARY)(filePath)}`, true)
+      konsola.ok(`Successfully removed entry for machine ${machineName} from ${chalk.hex(colorPalette.PRIMARY)(filePath)}`, true)
     }
-  }
-  catch (error: unknown) {
-    handleFileSystemError('write', error as NodeJS.ErrnoException)
-  }
-}
-
-export function removeAllNetrcEntries(filePath = getNetrcFilePath()) {
-  try {
-    return writeFile(filePath, '', {
-      mode: 0o600, // Set file permissions
-    })
-  }
-  catch (error) {
-    handleFileSystemError('write', error as NodeJS.ErrnoException)
-  }
-}
-
-export async function isAuthorized() {
```

Comment:
this and `removeAllNetrcEntries` are imported and used in `src/commands/logout/index.ts`